make/targets/golang/imagebuilder.mk

```diff
@@ -0,0 +1,30 @@
+self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
```

Comment:
This file should rather live under `make/targets/openshift` and call it just `imagebuilder.mk`

Reply:
Ack, will do.